pom.xml

```diff
@@ -19,8 +19,8 @@
   <properties>
     <versionString>12.0</versionString>
     <xccVersion>${versionString}</xccVersion>
-    <hadoopVersion>3.3.4</hadoopVersion>
-    <jacksonVersion>2.14.0-rc3</jacksonVersion>
+    <hadoopVersion>3.3.6</hadoopVersion>
+    <jacksonVersion>2.15.2</jacksonVersion>
```

Comment:
What vulnerability does jackson have?

Reply:
It has a weakness.. https://cwe.mitre.org/data/definitions/400.html